metrics/time/launch_times.sh

```diff
@@ -38,6 +38,34 @@ REQUIRED_CMDS=("bc" "awk")
 # Note the no-0-padding - 0 padding the results breaks bc in some cases
 DATECMD="date -u +%-s:%-N"
 
+# This the minimum entropy level produced
+# by haveged is 1000 see https://wiki.archlinux.org/index.php/Haveged
+# Less than 1000 could potentially slow down cryptographic
+# applications see https://www.suse.com/support/kb/doc/?id=7011351
+entropy_level="1000"
+
+check_entropy_package() {
```

Comment:
I'm not sure we should do a hard check for a specific entropy daemon. People might be running this test via the metrics report generator on a distro or setup that uses a different entropy source. This is not just for our CI systems.
We probably don't need to check there is entropy at all. What we can do is maybe have a timeout on waiting for entropy - if we don't get enough entropy in say 10s, then make the test fail saying 'not enough entropy to perform a reliable test' for instance.